components/typeform/sources/new-submission/new-submission.mjs

```diff
@@ -15,58 +18,54 @@ function parseIsoDate(isoDate) {
 }
 
 export default {
+  ...common,
   key: "typeform-new-submission",
   name: "New Submission",
-  version: "0.0.4",
+  version: "0.0.5",
   type: "source",
   description: "Emit new submission",
   props: {
     typeform,
```

Comment:
I think we would typically [inherit/extend](https://github.com/PipedreamHQ/pipedream/blob/9789bd637e61143a5d980b35cff12758fcee5513/docs/docs/components/guidelines/README.md#common-files-optional) the `props`, `methods`, and `hooks` [properties](https://pipedream.com/docs/components/api/#component-structure) from the base/common file, when overriding those component properties (even if they're not defined in the common file).